### PR TITLE
PR: changed hint line

### PR DESCRIPTION
**Pull request by Mastercodester**
deleted the word extras

### CODE REVIEW

File: MacOSSwiftUI/MacOSSwiftUI/ContentView.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ struct ContentView: View {
             List {
                 ForEach(items) { item in
                     NavigationLink {
-                        Text("Item located at \(item.timestamp!, formatter: itemFormatter)").accessibilityHint("This is some extra Info")
+                        Text("Item located at \(item.timestamp!, formatter: itemFormatter)").accessibilityHint("This is some ")
```
Comment (BhushanAbhyankar) [inline MacOSSwiftUI/MacOSSwiftUI/ContentView.swift]: Please add something meaningful info for the hint

Comment (BhushanAbhyankar) [inline MacOSSwiftUI/MacOSSwiftUI/ContentView.swift]: Dont use force unwrap for optionals for item.timestamp

**Review by BhushanAbhyankar — CHANGES_REQUESTED** [review]
Do asked changes

File: MacOSSwiftUI/MacOSSwiftUI/ContentView.swift (hunk 1 above)
Comment (Mastercodester) [reply MacOSSwiftUI/MacOSSwiftUI/ContentView.swift]: I have added an meaningful hint

Comment (Mastercodester) [reply MacOSSwiftUI/MacOSSwiftUI/ContentView.swift]: I have removed the force unwrap and given default value

**Review by BhushanAbhyankar — APPROVED** [review]
New Changes looks Good. We are ok to merge this PR

**Pull request merged**